src/window.rs

```diff
@@ -521,7 +521,7 @@ impl Window {
     ///
     /// - **iOS:** Can only be called on the main thread. Returns the `PhysicalSize` of the window's
     ///   [safe area] in screen space coordinates.
-    /// - **Web:** Returns the size of the canvas element.
+    /// - **Web:** Returns the size of the canvas element's framebuffer.
```

Comment:
A comment regarding documentation, `inner_size`'s documentation for web talks about the "framebuffer size", but `set_inner_size` talks about the "content box via the `style` attribute". However, if we have HiDPI (device pixel ratio > 1), the framebuffer size (physical size) will be bigger than the style attribute (logical size). So the documentation looks inconsistency to me, for two functions that look like getter and setter from their names. Or am I missing something?

Reply:
Sort of. `inner_size` returns the framebuffer size as a typed `PhysicalSize`, which when converted to a `LogicalSize` using `PhysicalSize::to_logical` will be ~equal to the logical size set with `set_inner_size`.

It's true that they're not a proper getter/setter pair though. It's possible for the CSS set by `set_inner_size` to be overridden with `!important`, causing `set_inner_size` to have no effect  what's returned by `inner_size`.

Reply:
Ah yes, I missed the return type. Thank you for the explanation! An option could be for `set_inner_size` to be called something like `set_inner_logical_size` to account for its parameter to be untyped wrt physical vs logical. But it is of course a tradeoff between naming simplicity and semantic precision...